userspace/libscap/examples/01-open/scap_open.c

```diff
@@ -760,40 +760,62 @@ void print_stats()
 {
 	gettimeofday(&tval_end, NULL);
 	timersub(&tval_end, &tval_start, &tval_result);
+	uint32_t flags = PPM_SCAP_STATS_KERNEL_COUNTERS | PPM_SCAP_STATS_LIBBPF_STATS;
+	uint32_t nstats;
+	int32_t rc;
+	const scap_stats_v2* stats_v2;
+	stats_v2 = scap_get_stats_v2(g_h, flags, &nstats, &rc);
+	const scap_machine_info* scap_machine_info = scap_get_machine_info(g_h);
+	/* Current contract: `n_evts` always first stats item. */
```

Comment:
Is this contract specified anywhere except this comment? Also, why do we need it?